SPECS/grep/grep.spec

```diff
@@ -12,7 +12,6 @@ BuildRequires:  pcre2-devel
 %if %{with_check}
 BuildRequires:  perl(File::Find)
 %endif
-Requires:       pcre2
```

Comment:
don't you need to bump release nb ?

Reply:
no, since the package hasn't yet been published.